services/core-api/app/api/mines/permits/permit_conditions/resources/permit_amendment_condition_category_user_resource.py

```diff
@@ -1,19 +1,45 @@
 from flask_restx import Resource, reqparse
-from werkzeug.exceptions import BadRequest, NotFound
+from werkzeug.exceptions import BadRequest, NotFound, Unauthorized
 
 from app.api.mines.permits.permit_conditions.models import PermitConditionCategory
-from app.api.mines.response_models import PERMIT_CONDITION_CATEGORY_MODEL
+from app.api.mines.permits.permit_conditions.models.permit_condition_review_assignment import PermitConditionReviewAssignment
+from app.api.mines.permits.permit_amendment.models.permit_amendment import PermitAmendment
+from app.api.mines.response_models import PERMIT_CONDITION_CATEGORY_MODEL, PERMIT_CONDITION_REVIEW_ASSIGNMENT_MODEL
 from app.api.users.models.user import User
 from app.api.utils.access_decorators import requires_role_edit_standard_permit_conditions, \
     requires_role_view_all, EDIT_STANDARD_PERMIT_CONDITIONS
-from app.extensions import db, api, jwt
+from app.extensions import api, jwt
 from app.api.utils.include.user_info import User as UserUtils
 
 
 class AssignUserToPermitConditionCategory(Resource):
-    @api.doc(description='Assign a user to a permit condition category')
+    @api.doc(
```

Comment:
Alternative to adding `api.doc` here so we don't have to define it twice while still having swagger pick it up:

- Move `parser` out of the method and onto the class
- And do `@api.expects(parser)`

Reply:
I don't think I can in this case as the different endpoints take different parameters. 🤔